test_conformance/math_brute_force/main.cpp

```diff
@@ -194,6 +194,11 @@ int doTest( const char* name )
                 }
             }
         }
+        else
```

Comment:
This will print the message even if `func_data->relaxed` is false anyway.   I.e. it will print "skip" for tests that would never run anyway.

Reply:
Good point, I have updated this and refactored some of the control flow to make it cleaner.

Reply:
Thanks.